server/src/Infrastructure/HumanResource/Payslip/Action/GetUsersElementsCsvAction.ts

```diff
@@ -8,6 +8,9 @@ import { Response } from 'express';
 import { format } from 'date-fns';
 import { LeaveRequestSlotView } from 'src/Application/HumanResource/Leave/View/LeaveRequestSlotView';
 
+const formatFrNumber = (n: number) =>
+  n.toLocaleString('fr-FR').replaceAll(' ', '');
```

Comment:
En fait en utilisant un regexp ça remplace tout @benpaquier 

Ça permet d'utiliser un `\s` qui prend en compte tout type d'espace. Car avais-tu remarqué que j'avais ici pas mis un simple espace (code ASCII 32), mais un ǹarrow no-break space` (ce qui est utilisé par `toLocaleString('fr-FR')` pour la séparation entre les milliers) ? #hack

```suggestion
  n.toLocaleString('fr-FR').replace(/\s/g, '');
```